x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/change_password/10_basic.yml

```diff
@@ -83,6 +83,43 @@ teardown:
       cluster.health: {}
   - match: { timed_out: false }
 
+---
+"Test changing users password with prehashed password":
+  - skip:
+      features: ["catch_unauthorized", "warnings"]
+  - do:
+      headers:
+        Authorization: "Basic am9lOnMza3JpdC1wYXNzd29yZA=="
+      cluster.health: {}
+  - match: { timed_out: false }
+
+  # We need this so that we can get hold of the password hash of the user without conditionals or multiple tests as the default hashing
+  # algorithm is different in FIPS mode. Additionally, for bcrypt, the stored hash string starts with a "$" and it would otherwise be
+  # interpreted as a stashed value, if we tried to use it as the value of "password_hash".
+  - do:
+      warnings:
+        - "this request accesses system indices: [.security-7], but in a future major version, direct access to system indices will be prevented by default"
+      get:
+        index: .security
+        id: user-joe
+  - set: { _source.password: "hash" }
+
+  # We change the password to the existing one
+  - do:
+      security.change_password:
+        username: "joe"
+        body:  >
+          {
+            "password_hash" : "$hash"
+          }
```

Comment:
Sorry to be a pain, can we change it to something else (using normal change password) and then change it back to the hash rather than just relying on a no-op update?

Reply:
No pain, you're right, wiil take care of this